test/escort.test.js

```diff
@@ -15,1586 +15,1702 @@ var makeBadString = (function (Ctor) {
     };
 }(String));
 
-module.exports = {
-    "methods static": function () {
-        var url;
-        var app = connect(
-            escort(function (routes) {
-                url = routes.url;
-                
-                methods.forEach(function (method) {
-                    routes[method]("home_" + method, "/" + method, function (req, res) {
-                        res.end(method.toUpperCase() + " /" + method);
-                    });
-                });
-            })
-        );
-        
+function makeConnect(...apps) {
+  const c = connect();
+
+  for (let app of apps) {
+    c.use(app);
+  }
+
+  return c;
+}
+
+describe("escort", function() {
+  after(function() {
+    // apps leave references that don't get cleaned up
+    process._getActiveHandles()
```

Comment:
We create tons of `connect` apps which each add a listener; just get rid of them at the end so that the runloop can exit.